src/test/java/com/powsybl/openloadflow/NonImpedantBranchTest.java

```diff
@@ -379,7 +384,7 @@ void securityAnalysisTest() {
      *                   ld5
      */
     @Test
-    void securityAnalysisNotSameNumberOfVariablesAndEquationsIssueTest() {
```

Comment:
We create this test with this name because we spend long hours to solve these equation system issues with zero impedance branches. Please find a way to keep this information (at least in comments) or refactor to share the unit test.

Reply:
We put that test here because it is an equation system issue. Your test should be in the security analysis part, with a share of the network in a factory, and it will compact the code as we have dedicated methods to run security analysis and get the results.

Reply:
@annetill should be fixed now